README.md

```diff
@@ -340,6 +341,32 @@ end
 ```
 </details>
 
+## Testing
+
+To test controller that are protected by a `before_action :require_user`, require Passwordless's test helpers in your test suite.
+
+For `rspec`, add the following line to your `spec/rails_helper.rb` or
+`spec/spec_helper` if `rails_helper` does not exist:
+
+```ruby
+require "passwordless/test_helpers"
+```
+
+For `test-unit`, add this line to your `test/test_helper.rb`:
+
+```ruby
+require "passwordless/test_helpers"
+```
+
+
+Then in your controller, request, and system tests/specs, you can utilize the following methods:
+
+```ruby
+passwordless_sign_in # takes you to the sign in path
+passwordless_sign_in(user) # signs you in as a user
```

Comment:
I'm not sure I like this dual meaning API. Let's split it into `passwordless_sign_in(resource)` and `passwordless_sign_in_path`.